x/marker/types/params.go

```diff
@@ -60,6 +69,7 @@ func DefaultParams() Params {
 		DefaultMaxTotalSupply,
 		DefaultEnableGovernance,
 		DefaultUnrestrictedDenomRegex,
+		math.NewIntFromUint64(DefaultMaxSupply),
```

Comment:
The default value should be promoted to a `math.int` to better reflect the expanded range.